src/hotspot/share/gc/shared/gcLocker.cpp

```diff
@@ -136,6 +136,7 @@ void GCLocker::jni_lock(JavaThread* thread) {
     // in the code, but it's too strong; it's possible that the last thread
     // has called `jni_unlock`, but not yet finished the call, e.g. initiating
     // a GCCause::_gc_locker GC.
+    log_debug_jni("Blocked from entering critical section while waiting on GC.");
```

Comment:
Seems unrelated to current MR - leftover debugging code?

Reply:
Yes, good catch. Leftover from when this patch used GCLocker (which turned out to be not needed in the end)